src/types/hooks/index.ts

```diff
@@ -4,8 +4,8 @@ import type BigNumber from "bignumber.js";
 import type { BrowserProvider } from "ethers";
 
 export enum ConnectorNames {
-  METAMASK = "MetaMask",
-  CRYPT_KEEPER = "CryptKeeper",
+  METAMASK = "Metamask",
```

Comment:
I'd say let's keep it the same. `MetaMask` and `CryptKeeper`
Because those are the names shown in the store for both of them.